tanner/config.py

```diff
@@ -9,7 +9,9 @@
                             'vdocs': '/opt/tanner/data/vdocs.json'},
                    'TANNER': {'host': '0.0.0.0', 'port': 8090},
                    'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
-                   'EMULATORS': {'root_dir': '/opt/tanner'},
+                   'EMULATORS': {'root_dir': '/opt/tanner',
+                                  'emulator_enabled': {'sqli': True, 'rfi': True, 'lfi': True, 'xss': True, 'cmd_exec': True}
```

Comment:
Could we store only emulators names as list? And if emulator in the list, it's enabled?